tg_bot/__main__.py

```diff
@@ -415,6 +415,34 @@ def migrate_chats(bot: Bot, update: Update):
     LOGGER.info("Successfully migrated!")
     raise DispatcherHandlerStop
 
+def is_chat_allowed(bot, update):
+    if len(WHITELIST_CHATS) =! 0:
+        chat_id = update.effective_message.chat_id
+        if chat_id not in WHITELIST_CHATS:
+            bot.send_message(chat_id=update.message.chat_id,
```

Comment:
if you used effective_message earlier, you should use effective_message now. Stay consistent.